lib/puppet/indirector/hiera.rb

```diff
@@ -63,7 +63,7 @@ def convert_merge(merge)
         convert_merge(strategy)
       end
     else
-      raise Puppet::DataBinding::LookupError, "Unrecognized value for request 'merge' parameter: '#{merge}'"
+      raise Puppet::DataBinding::LookupError, _("Unrecognized value for request 'merge' parameter: '%{merge}'") % { merge: merge }
```

Comment:
Could you add a translator note that `merge` is an application parameter (not to be translated)?